src/lib/classes/lsp3-universal-profile.spec.ts

```diff
@@ -230,11 +235,11 @@ describe('LSP3UniversalProfile', () => {
 
     describe('Deployment with only ERC725 baseContract set to true', () => {
       it('Should deploy only ERC725 Base contract', async () => {
-        await testUPDeploymentWithBaseContractFlag(
+        await testUPDeployment(
           {
-            ERC725Account: true,
-            KeyManager: false,
-            UniversalReceiverDelegate: false,
+            ERC725Account: { deployProxy: true },
```

Comment:
Here you could do the opposite of my comment above.

Or for instance (but not fully sure), the test could be:
1) extract the address of the base contract inside the proxy bytecode.
2) call this contract and check that it supports the `LSP0ERC725Account` interface. Like `baseContract.supportsInterface("0x63cb749b")`.

See list of interface IDs here: https://docs.lukso.tech/standards/smart-contracts/interface-ids